http4k-security/oauth/src/main/kotlin/org/http4k/security/oauth/client/clientFilterExtensions.kt

```diff
@@ -67,16 +70,18 @@ fun ClientFilters.RefreshingOAuthToken(
 fun ClientFilters.RefreshingOAuthToken(
     config: OAuthProviderConfig,
     backend: HttpHandler,
-    oAuthFlowFilter: Filter = ClientFilters.OAuthClientCredentials(config.credentials),
+    oAuthFlowFilter: Filter = ClientFilters.OAuthClientCredentials(config.credentials, emptyList()),
     gracePeriod: Duration = Duration.ofSeconds(10),
-    clock: Clock = Clock.systemUTC()
+    clock: Clock = Clock.systemUTC(),
+    scopes: List<String> = emptyList(),
```

Comment:
I guess it generally makes also sense to have an option to supply the scope for `ClientFilters.OAuthCredentials()`.
The problem is that the scope has to be provided again at for `ClientFilters.RefreshingOAuthToken`.
We could move `scopes` up in the parameter list, but this would introduce a breaking change.

Reply:
For now I went with highlighting that no scopes are passed into the oAuthFlowFilter - this way it's not breaking.
What do you think?